JacksonDunstanNativeCollections/NativeElement.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using Unity.Collections;
+using Unity.Collections.LowLevel.Unsafe;
+
+/// <summary>
+/// NativeElement is a wrapper for a single blittable element to be passed between managed and unmanaged code in jobs
+/// </summary>
+/// <typeparam name="T">The blittable type to be stored</typeparam>
+[StructLayout(LayoutKind.Sequential)]
+[NativeContainer]
+[DebuggerDisplay("{Element}")]
```

Comment:
Add `[NativeContainerSupportsDeallocateOnJobCompletion]` for consistency with `NativeIntPtr` and to allow this feature.